index.bs

```diff
@@ -8447,8 +8462,8 @@ at most one interface.  The interface [{{PrimaryGlobal}}]
 is declared on, if any, is known as the <dfn id="dfn-primary-global-interface" export>primary global interface</dfn>.
 
 See [[#named-properties-object]],
-[[#getownproperty]] and
-[[#defineownproperty]]
+[[#legacy-platform-object-getownproperty]] and
```

Comment:
These two references are no longer relevant, actually, now that globals have been decoupled.